client/src/utils/election.ts

```diff
@@ -16,6 +22,54 @@ export const getContests = ({
       ballotStyle.partyId === c.partyId
   )
 
+export const getEitherNeitherContests = (
+  contests: Contests
+): MsEitherNeitherContest[] =>
+  contests
+    .filter((c) => c.type === 'ms-either-neither')
+    .map((c) => c as MsEitherNeitherContest)
```

Comment:
```suggestion
  contests.filter(
    (c): c is MsEitherNeitherContest => c.type === 'ms-either-neither'
  )
```

If `filter` is given a type predicate the return value will be inferred correctly.